charts/stable/fireflyiii/questions.yaml

```diff
@@ -73,7 +73,21 @@ questions:
                     - value: "OnDelete"
                       description: "(Legacy) OnDelete: ignore .spec.template changes"
 # Include{controllerExpert}
-  # Docker specific env
+  - variable: secret
+    group: "Container Configuration"
+    label: "Image Secrets"
+    schema:
+      type: dict
+      attrs:
+        - variable: STATIC_CRON_TOKEN
+          label: "Static CRON token"
+          description: ""
+          schema:
+            type: string
+            default: ""
+            min_length: 32
+            max_length: 32
+            valid_chars: '[a-zA-Z0-9!@#$%^&*?]{32}'
```

Comment:
Nope use automatic remembered generation, like we use for all the database passwords and such.

Reply:
Okie! Once I figure why it's not working as it should with the static token i'll refactor it

Reply:
Cool!

Reply:
Oh, I figured it out.. We have an outdated container because of the official repo change 5 months ago :)
new repo is under `fireflyiii/core`
https://github.com/firefly-iii/firefly-iii/releases/tag/5.5.12

Reply:
We are in the middle of a god-damn rework.